src/main/java/com/elastic/support/scrub/ScrubApp.java

```diff
@@ -0,0 +1,29 @@
+package com.elastic.support.scrub;
+
+import com.elastic.support.config.ScrubInputs;
+import com.elastic.support.util.JsonYamlUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+
+
+public class ScrubApp {
+
+    private static Logger logger = LogManager.getLogger(ScrubApp.class);
+
+    public static void main(String[] args) {
+
+        ScrubInputs scrubInputs = new ScrubInputs();
+        scrubInputs.parseInputs(args);
+        if(!scrubInputs.validate()){
+            logger.info("Exiting...");
+            System.exit(0);
```

Comment:
Same suggestion. Perhaps

```suggestion
            System.exit(400);
```

Reply:
In this case, the only reason it would exit is the lack of a required parameter - and the JCommander would be spitting out the message. Given that in this case I'm initiating an orderly termination I didn't see the point of making it seem like something dire. Java exit codes usually don't mean anything anyway - the question would be if there were something else running that was monitoring non zero codes and seeing it as something worse than a bad parameter. Don't know, but not sure what a 400 gets you here given nothing is making it past the initial main method?

Reply:
> Don't know, but not sure what a 400 gets you here given nothing is making it past the initial main method?

Just that any shell would notice the non-zero exit code and could log something to standard error for it.